### ISSUE TITLE
Wrong errors reported in VSCode

### ISSUE THREAD

**Issue by lobo-tuerto** [pantajoe/vscode-elixir-credo#465]
I have set up a new Phoenix project, added credo as a dependency, then installed this VSCode extension to get info/hints on code style violations.

But, inside VSCode I get a squiggly line on every `defmodule` with a false positive, for example:

```bash
Test files should end with .exs (warning:Credo.Check.Warning.WrongTestFileExtension)
Simulation.Core.Vector2D
```

Which in itself is wrong, since that's not a test file (`lib/simulation/core/vector_2d.ex`).
I've tried restarting the extension host, and the Elixir language server but the spurious report persist.

Running `mix credo --strict` from the CLI does not yield those errors, so I think it's related to the extension or configuration maybe?

Any recommendations or pointers on what could be wrong, or how to debug what's going on?

Cheers,
VA

**Comment by lobo-tuerto** [comment]
Additionally, in test files I get these:

```bash
Test files should end with .exs (warning:Credo.Check.Warning.WrongTestFileExtension)
Modules should have a @moduledoc tag. (readability:Credo.Check.Readability.ModuleDoc)
Simulation.Core.Vector2DTest
```

Those are for: `test/simulation/core/vector_2d_test.exs`.

Which both seem wrong. :thinking:

**Comment by reid-rigo** [comment]
I'm seeing this as well. It appears to be caused by #464

**Comment by Morgahl** [comment]
This has been addressed in the `credo` source repo as of `1.7.5` if you want to step your version min.

https://github.com/rrrene/credo/issues/1115

**Comment by pantajoe** [comment]
Since there's no further activity since, I'll assume that 1.7.5 solved this issue and after addressing #464 this issue would be a duplicate, therefore, I'll close this one.
